hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/timeAbsence.jsp

```diff
@@ -88,25 +88,25 @@
             <table class="dl-table table">
               <thead>
                 <tr rsf:id="header:">
-                  <th class="flc-pager-sort-header" rsf:id="name"><a href="javascript:;">Name</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="status"><a href="javascript:;">Status</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="start"><a href="javascript:;">Start</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="end"><a href="javascript:;">End</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="total"><a href="javascript:;">Total</a></th>
+                  <th scope="col" class="flc-pager-sort-header" rsf:id="name"><a href="#">Name</a></th>
```

Comment:
I was only be able to test it in the inspector of browsers. it works.
could not test in local.

On Thu, Aug 13, 2015 at 2:08 PM, Tim Levett <notifications@github.com>
wrote:

> In hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/timeAbsence.jsp
> <https://github.com/UW-Madison-DoIT/hrs-portlets/pull/38#discussion_r37013203>
> :
>
> > @@ -88,25 +88,25 @@
> >              <table class="dl-table table">
> >                <thead>
> >                  <tr rsf:id="header:">
> > -                  <th class="flc-pager-sort-header" rsf:id="name"><a href="javascript:;">Name</a></th>
> > -                  <th class="flc-pager-sort-header" rsf:id="status"><a href="javascript:;">Status</a></th>
> > -                  <th class="flc-pager-sort-header" rsf:id="start"><a href="javascript:;">Start</a></th>
> > -                  <th class="flc-pager-sort-header" rsf:id="end"><a href="javascript:;">End</a></th>
> > -                  <th class="flc-pager-sort-header" rsf:id="total"><a href="javascript:;">Total</a></th>
> > +                  <th scope="col" class="flc-pager-sort-header" rsf:id="name"><a href="#">Name</a></th>
>
> So in testing the application post your fix, did the sorting still work?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UW-Madison-DoIT/hrs-portlets/pull/38/files#r37013203>.
>